src/base/bittorrent/session.cpp

```diff
@@ -299,7 +299,8 @@ Session::Session(QObject *parent)
     , m_maxUploads(BITTORRENT_SESSION_KEY("MaxUploads"), -1, lowerLimited(0, -1))
     , m_maxConnectionsPerTorrent(BITTORRENT_SESSION_KEY("MaxConnectionsPerTorrent"), 100, lowerLimited(0, -1))
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
-    , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
+    , m_btProtocol(BITTORRENT_SESSION_KEY("BTProtocol"), BTProtocol::Both
+        , clampValue(BTProtocol::Both, BTProtocol::UTP))
```

Comment:
Not related to this PR, with this function name (`clampValue`) the code is a bit confusing to read. Wish it would have name `make_clapper` or similar.

Reply:
@evsh:  

>`make_clapper` 

And what is this `clapper` supposed to be?  

(Google was no help at all.)

Reply:
>And what is this clapper supposed to be?

I don't about that but `clampValue()` takes 2 arguments. The min and max of a range of values. Then it compares user value. If the user value is out of range it reassigns to it the min/max value.